pandas/_libs/khash.pxd

```diff
@@ -17,6 +17,7 @@ from numpy cimport (
 
 cdef extern from "khash_python.h":
     const int KHASH_TRACE_DOMAIN
+    void KHASH_SET_INIT_INT64(int64)
```

Comment:
This is not needed here. IIUC `KHASH_SET_INIT_INT64 is not a function but a macro

Reply:
Yea I'll remove this. Leftover from some experimentation